apps/cowswap-frontend/src/modules/twap/containers/TwapConfirmModal/index.tsx

```diff
@@ -83,6 +82,7 @@ export function TwapConfirmModal({ fallbackHandlerIsNotSet }: TwapConfirmModalPr
         isConfirmDisabled={isConfirmDisabled}
         priceImpact={priceImpact}
         buttonText={'Place TWAP order'}
+        refreshInterval={PRICE_UPDATE_INTERVAL}
```

Comment:
@shoom3301 FYI, just in case it's not fixed in a later PR
![image](https://github.com/cowprotocol/cowswap/assets/43217/d20769fc-700d-4ee6-a36d-2b6dfca9dabf)